tools/fix_auth/models.rb

```diff
@@ -87,21 +87,24 @@ def self.contenders
 
   class FixDatabaseYml
     attr_accessor :id
-    attr_accessor :yaml
+    attr_accessor :yml
     include FixAuth::AuthConfigModel
 
     class << self
       attr_accessor :available_columns
       attr_accessor :file_name
-      alias_method  :table_name, :file_name
+
+      def table_name
+        file_name.gsub(".yml", "")
```

Comment:
Perhaps `File.basename(file_name, ".yml")`? Not totally sure if it works here, though, since I'm not sure where the extension can appear within the overall filename.

Reply:
thnx.

I wanted to include the full path of the filename.
Besides that they seemed to do the same thing.